pkg/alertmanager/operator.go

```diff
@@ -57,6 +58,7 @@ import (
 
 const (
 	resyncPeriod = 5 * time.Minute
+	OperatorName = "alertmanageroperator"
```

Comment:
I was thinking that we could have consistent telemetry fields here. In this example, we use alertmanageroperator as the value for the `component` field in all logs from this controller.

I agree that, when looking at events, it will be weird to see "alertmanageroperator", but I already find it weird seeing it in the logs 😅

Reply:
If you have a strong opinion on keeping "prometheus-operator" for all events I totally get it and can do the change ASAP. Let me know :)

Reply:
Also from the point of view of a user provisioning ServiceMonitor, AlertmanagerConfig, ..., it doesn't really matter which indivitual controller rejected the resource.

Reply:
At least I wouldn't use "alertmanager-operator" but "alertmanager-controller" to avoid the confusion.

Reply:
> I would be in favor of changing the logs to have a "controller=alertmanager" key/val which would be consistent with the controller metrics (e.g. `prometheus_operator_ready`).

Sounds good, changing label values in metrics is more of a breaking change than changing logs fields. Let's open an issue for making those fields consistent across logs and metrics?